src/templates/mixins/shape.pug

```diff
@@ -5,4 +5,7 @@ mixin shape(name, title)
 		if title
 			title= title
 
-		use(xlink:href='#shape-' + name)
+		use(xlink:href=`${assetPath}/sprites/shapes.svg#shape-${name}`)
+
+		//- IE fallback
+		use(xlink:href=`#shape-${name}`)
```

Comment:
The polyfill code should handle this case (replace all occurences of en external resource to local one) and it should not be included in the HTML twice.

Reply:
1. This way the code can be easily used in lazy loaded html content.
2. IE is very bad at handling high JavaScript load. Scanning a whole page can do unnecessary harm during the page load while other work needs to be done too.

Reply:
@enzy What do you think? Is the double `use` justified?